test/cli_labrc/.labrc

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
I agree, would be good to use an extension. Eslint deprecated `.eslintrc` too in favor of `.eslintrc.json`, `.eslintrc.yaml `, etc.